packages/core/src/test/volume-prop.test.ts

```diff
@@ -0,0 +1,45 @@
+import {evaluateVolume} from '../volume-prop';
+import {expectToThrow} from './expect-to-throw';
+
+describe('EvaluateVolume does not throw', () => {
+	test('Volume is undefined', () => {
+		const toEvaluate = {frame: 10, volume: undefined};
+		expect(evaluateVolume(toEvaluate)).toEqual(1);
+	});
+	test('Volume is smaller than one', () => {
+		const smallVolume = 0.5;
+		const toEvaluate = {frame: 10, volume: smallVolume};
+		expect(evaluateVolume(toEvaluate)).toEqual(smallVolume);
+	});
+	test('Volume is bigger than one', () => {
+		const toEvaluate = {frame: 10, volume: 10};
+		expect(evaluateVolume(toEvaluate)).toBe(1);
+	});
+	test('evaluated volume from frame that is smaller than one', () => {
+		const smallFrame = 0.5;
+		const toEvaluate = {frame: smallFrame, volume: (frame: number) => frame};
+		expect(evaluateVolume(toEvaluate)).toBe(smallFrame);
```

Comment:
Testing a case that should never happen, `frame` will always be an integer.

I am changing it to frame * 0.5 which should be the same